src/Framework/PaymentGateways/DataTransferObjects/GatewayRouteData.php

```diff
@@ -24,13 +24,18 @@ class GatewayRouteData
      * @var string|null
      */
     public $routeSignature;
+    /**
+     * @var string|null
+     */
+    public $routeSignatureId;
 
     /**
      * Convert data from request into DTO
      *
-     * @return self
+     * @since 2.19.4 - add give-route-signature-id
```

Comment:
Small thing, but the `-` separator does not match the standard format. The format should be `@since x.x.x This is what changed`.

Reply:
resolved [991687db](https://github.com/impress-org/givewp/commit/991687dbae73c56ac6a1c030c3503238502a408d)